# HG changeset patch
# User Hannu Savolainen <hannu@opensound.com>
# Date 1329431121 -7200
# Node ID 1a5c5373b72a7a197819d6da0f220f49be350577
# Parent  0500de7fee6f2c9baf0b241db37c5a6864e543ce
Fixed reporting of AFMT_FLOAT as a supported format

diff -r 0500de7fee6f -r 1a5c5373b72a kernel/framework/vmix_core/vmix_core.c
--- a/kernel/framework/vmix_core/vmix_core.c	Tue Feb 07 05:25:23 2012 +0200
+++ b/kernel/framework/vmix_core/vmix_core.c	Fri Feb 17 00:25:21 2012 +0200
@@ -1691,8 +1691,11 @@
 						&vmix_driver,
 						sizeof (audiodrv_t),
 						opts,
-						AFMT_S16_NE | AFMT_S32_NE |
-						AFMT_FLOAT, mixer, -1)) < 0)
+#ifdef CONFIG_OSS_VMIX_FLOAT
+						AFMT_FLOAT |
+#endif
+						AFMT_S16_NE | AFMT_S32_NE,
+						mixer, -1)) < 0)
     {
       return portc->audio_dev;
     }
